test/test_distributions.py

```diff
@@ -434,8 +439,9 @@ def test_constraints(constraint, x, expected):
     assert_array_equal(constraint(x), expected)
 
 
-@pytest.mark.parametrize('shape', [(), (1,), (3,), (5,), (3, 1), (1, 3), (5, 3)], ids=idfn)
+@pytest.mark.parametrize('shape', [(), (1,), (3,), (6,), (3, 1), (1, 3), (5, 3)], ids=idfn)
```

Comment:
We change this to 6 to make sure that it is the length of a lower tril vector (of 4 x 4 matrix)